tests/fuzz/Makefile.am

```diff
@@ -43,6 +43,7 @@ AM_CPPFLAGS                                               = \
 COMMON_LDADD                                              = \
     $(top_builddir)/src/core/libopenthread-ftd.a            \
     $(top_builddir)/third_party/mbedtls/libmbedcrypto.a     \
+	$(top_builddir)/third_party/tcplp/libtcplp.a            \
```

Comment:
nit: use spaces instead of tab
```suggestion
    $(top_builddir)/third_party/tcplp/libtcplp.a            \
```

Reply:
Thanks for finding this. I've addressed this now in the latest version.